setup.cfg

```diff
@@ -24,6 +24,23 @@ install_requires =
     pyyaml
     softioc
 
+[options.extras_require]
+# For development tests/docs
+dev = 
+    black==19.10b0
```

Comment:
Can you keep the version of black that we currently have please? The only reason I left it in the template module was so I didn't have to make any code changes there...